### ISSUE TITLE
Compare GTFS to NJTPA shapefile

### ISSUE THREAD

**Issue by sromalewski** [BetaNYC/NY-Waterways-GTFS-data#1]
Chris, this data looks very intriguing (and kudos to Noel for requesting it). But I converted the "shapes" file to a line representation of the routes, and it displays a very crude map of ferry routes (and it's not clear to me that the end points of the lines - which should be at the ferry terminals themselves - are spatially accurate).

In contrast, the North Jersey Transportation Planning Authority provides a shapefile of NJ ferry routes to NYC, and the route lines look nice and smooth and, more important, accurately drawn. See http://geoportal.njtpa.org:8080/geoportal/catalog/search/viewMetadataDetails.page?uuid=%7B4E63C8FA-29A4-4655-8CBE-3B2474A3E586%7D

I wonder if you've come across a comprehensive GIS data set(or sets) of ferry routes in and around NY Harbor.  The NYC DOT ferry page lists lots of them http://www.nyc.gov/html/dot/html/ferrybus/ferintro.shtml but the routes are mostly private and at best the operators have PDF maps of the routes (which are sometimes only schematic representations of the routes (such as http://www.nywaterway.com/UserFiles/Files/nyw_ferry_edgewater_to_midtown-39th.pdf). 

Any suggestions would be greatly appreciated.  Thanks.

Steve

**Comment by chriswhong** [comment]
AFAIK, this was provided directly by the NY Waterway Ferry, but it's good to know if their shapes.txt doesn't look accurate.  

I am not aware of any comprehensive GIS data for ferry routes... how many ferries are there?  Couldn't we just make one?

**Comment by sromalewski** [comment]
Yes, we can certainly make one - that's what we did about 10 years ago for the OASIS map (see the blue lines at http://bit.ly/1fmajCW).  Back then we were lucky enough to work with the director of the Metropolitan Waterfront Alliance, who was making a big push for ferry service funding and therefore needed a map showing what existed and what was proposed. They basically drew the lines for us, and we converted them to GIS format. Maybe we could ask them to help again.

There are a good number of lines/routes, so it'd take some focused effort.  According to NYC DOT, there are about four dozen routes that they list at http://www.nyc.gov/html/dot/html/ferrybus/ferintro.shtml  I'll reach out to DOT to see if they have a comprehensive shapefile of the routes.

The fallback is to just show ferry terminals without the routes, which we may end up doing.  DoITT has a shapefile at https://data.cityofnewyork.us/Transportation/Ferry-Terminal-Locations/9a56-tyfg and the DOT link above has the location info for each terminal.

We're hoping to hire a grad student soon for various projects, so maybe he/she can do the digitizing work for the routes.

**Comment by chriswhong** [comment]
I had no idea there were so many routes.  Do you need this for a specific
project, or just for general reference.  Maybe we should convene a
digitize-a-thon.
